package.json

```diff
@@ -136,6 +136,7 @@
     "size-limit": "^6.0.4",
     "ts-jest": "28.0.8",
     "ts-node": "10.9.1",
+    "tsd": "^0.28.1",
```

Comment:
what is that?

Reply:
I introduced this package in order to test types, see this: 

```ts
import { expectType } from 'tsd';

it('should infer state from initial state', () => {
  const { component } = setupStatefulComponent(initialState({ count: 0 }));
  expectType<RxState<{ count: number }>>(component.state);
});

it('should infer state from generic type', () => {
  const { component } = setupStatefulComponent<{ count: number }>();
  expectType<RxState<{ count: number }>>(component.state);
});
```

I think we can benefit from it in other places.